src/App.js

```diff
@@ -11,6 +11,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Landing />} />
         <Route path="/home" element={<Home />} />
+        <Route path="/trash" element={<Trash />} />
       </Routes>
     </div>
```

Comment:
suggestion,you can make folder for routes

Reply:
Sorry. I did not understand. folder for routes means? Do you mean a component where all routes will be present with `<Route/>` tags and use that component here?

Reply:
yes

Reply:
Nice idea. I will do it if time permits. thanks for the suggestion.